lightning/src/ln/peer_handler.rs

```diff
@@ -366,9 +367,10 @@ impl Peer {
 }
 
 struct PeerHolder<Descriptor: SocketDescriptor> {
-	peers: HashMap<Descriptor, Peer>,
-	/// Only add to this set when noise completes:
-	node_id_to_descriptor: HashMap<PublicKey, Descriptor>,
+	// Peer is under its own mutex for sending and receiving bytes, but note that we do *not* hold
```

Comment:
perhaps add a little blurb regarding what the actual purpose of this particular mutex is when doing i/o?

Reply:
In fact, the comment is stale, I corrected it.